client/lib/load-script/README.md

```diff
@@ -4,8 +4,9 @@ Async Script Loader
 This utility function allows us to use a standardized method of loading remote scripts and injecting them into the `<head>` of our document to bring external functionality into the app.
 
 ### Usage
-```js
-loadScript = require( 'lib/load-script' );
+```es6
+import loadScript from 'lib/load-script';
```

Comment:
Ahh sorry I wasn't clearer, I mean `loadScript.loadjQueryDependentScript` specifically (GitHub wouldn't let me select that line since it wasn't edited)